webpack.config.prod.js

```diff
@@ -87,6 +87,13 @@ module.exports = {
       template: './index.html',
     }),
 
+    // Set NODE_ENV to production so that Uglify can strip out dev-only code
```

Comment:
Cc @mzgoddard to review but I think we have to set it like this for webpack to remove the code. I can assert that this change has an impact where the package script one does not.